packages/neon_framework/pubspec.yaml

```diff
@@ -88,3 +88,10 @@ flutter:
       web:
         pluginClass: WebNeonPlatform
         fileName: src/platform/web.dart
+
+dependency_overrides:
+  package_info_plus:
+    git:
+      url: https://github.com/fluttercommunity/plus_plugins
+      path: packages/package_info_plus/package_info_plus
+      ref: baa336fe79c29411542fc343d332846bcd7752c1
```

Comment:
This should appear in the pubspec.lock

Reply:
ok `wakelock_plus` a dependency of `neon_news` was also depending on `package_info_plus`.
For now I just added the second override to the app itself as neon_news does not depend on web in any way.

If you want I can move it to news but I fear we forget to remove it from there afterwards.